### PR TITLE
[EA Forum only] post footer updates

### PR DESCRIPTION
**Pull request by s-cheng**
The final part of the "share posts" project: updating the post footer, to make the voting row shorter and include some of the icons from the post header. [Figma here](https://www.figma.com/file/d09iVzNSpZp9GyWGcfkWmX/Q2.2-2023?node-id=12%3A10488&mode=dev)

Screen Shot 2023-06-22 at 5 06 32 PM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204886218653310) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx [hunk 1]
```diff
@@ -115,6 +115,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     fontSize: theme.typography.body1.fontSize,
   },
   feedName: {
+    fontSize: theme.typography.body2.fontSize,
```
Comment (s-cheng) [inline packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:118]: Also reverted this font size to a slightly smaller one, based on [this slack thread](https://cea-core.slack.com/archives/CJY3ZKP62/p1687398875474129)

File: packages/lesswrong/components/votes/PostsVote.tsx [hunk 2]
```diff
@@ -46,8 +61,9 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 })
 
-const PostsVote = ({ post, classes }: {
+const PostsVote = ({ post, useHorizontalLayout, classes }: {
   post: PostsWithVotes,
+  useHorizontalLayout?: boolean, // if true, display the vote arrows to the left & right of the score
```
Comment (jpaddison3) [inline packages/lesswrong/components/votes/PostsVote.tsx:66]: This should be a `/** docstring */` instead for maximum communication

**Review by jpaddison3 — APPROVED** [review]
Very nice. I really like this change (cc @agnesstenlund). Big improvement.

**Comment by agnesstenlund** [comment]
I really like this too! 👏  looks almost perfect, just have some nit-picks:

- [ ] there are a few cases where it now looks a bit more crowded, like when there are mentions (here I'd probably want more padding around the mentions and also above the new upvote/share bottom bar (between the line and the post text))

Screenshot 2023-06-26 at 11 45 19


- [ ] Also looks like the divider for footnotes is a slightly different color + the footnotes now get different padding up and down:

Screenshot 2023-06-26 at 11 44 02

- [ ] Also there is this one post "which world gets saved" that shows the upvoting live, but not on localhost (and on localhost has no padding or line divider making it look a little crowded/ like mentions are a part of the text)

Localhost:
Screenshot 2023-06-26 at 11 47 09

Real forum:
Screenshot 2023-06-26 at 11 46 29

**Comment by s-cheng** [comment]
@agnesstenlund Thanks for catching those! I updated the spacing and the border color to match how the footnotes section works (though the border color is now a little darker and doesn't match the post header border, so maybe we want to just use the lighter color for all three of those lines).

Screen Shot 2023-06-27 at 6 38 16 PM

----- 
I found "which world gets saved" on localhost with prod db and it looks right to me, perhaps you were looking at the dev version so there were some discrepancies? 🤷‍♀️ 

Screen Shot 2023-06-27 at 6 39 33 PM

**Comment by s-cheng** [comment]
@agnesstenlund Ah I looked at the dev version of the post and that's what you were seeing. You're right that the bottom feels squished so I added some margin under the post body:

Screen Shot 2023-06-27 at 6 51 00 PM

**Pull request merged**